go/vt/vtctl/reparentutil/replication.go

```diff
@@ -307,9 +308,18 @@ func stopReplicationAndBuildStatusMaps(
 		}
 	}
 
+	numGoRoutines := len(tabletMap) - ignoredTablets.Len()
```

Comment:
Can you add some comments to the code block above? Lines 300 to 305 - to explain `numErrorsToWaitFor`.

Reply:
Will do 👍

Reply:
Done